.circleci/config.yml

```diff
@@ -0,0 +1,33 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/sd
+    docker:
+      - image: msheiny/pressfreedomci:latest
+        environment:
+          FPF_CI: true
+          CI_SD_ENV: staging
+          CI_AWS_TYPE: t2.small
+          FPF_GRSEC: false
+          TEST_REPORTS: /root/sd
+    steps:
+      # This is really dumb that Im doing this, basically
+      # I want to clone to ~/pressfreedom and work out of there
+      # the working_directory creates that directory first
+      - run: rm -r ~/sd
+      - checkout:
+          path: ~/sd
+      - run:
+          name: Installation pre-reqs
+          command: |
+            easy_install -U pip
+            pip install pip-tools
+            pip-sync testinfra/requirements.txt
```

Comment:
Say you have pypkg 1.0 installed. If you want to install a different version, regardless if it's an earlier or later version, you have to pass `-U`. It could be the case that you installed the right version with your `pip` command, but the preinstalled version was not installed with pip and is located under a different directory tree which comes earlier in the `PYTHONPATH`. Usually this isn't the case as where pip installs is before where package managers install in the `PYTHONPATH`.

As we've all experienced irregularities with `pip-sync` it seems best if you do take it out.